_posts/2016-05-18-splitAtFraction-method.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title:  "Dynamic work rebalancing & the splitAtFraction method"
+date:   2016-05-18 11:00:00 -0700
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+  - dhalperi
+---
+
+This morning, Eugene and Malo from the Google Cloud Dataflow team posted [*No shard left behind: dynamic work rebalancing in Google Cloud Dataflow*](https://cloud.google.com/blog/big-data/2016/05/no-shard-left-behind-dynamic-work-rebalancing-in-google-cloud-dataflow). This article discusses Cloud Dataflow’s solution to the well-known straggler problem.
+<!--more-->
```

Comment:
newline, so that it shows as 3 paragraphs total rather than the first two being merged (as it is in staging?)